archive/r/ruby/reverse-string.rb

```diff
@@ -0,0 +1,7 @@
+if ARGV.length < 1
+    puts "Usage: ruby reverse-string.rb [string]"
+end
+
+string = ARGV[0]
+
+puts string.reverse
```

Comment:
Can these bottom two lines go into an else case? I hadn't bothered printing usage before, but I like it! That said, this code errors out without a string.

Reply:
Fixed